java/update-related-features/README.md

```diff
@@ -1,22 +1,43 @@
-# Update Related Features
-A `MapView` loaded with two related `FeatureLayer`'s, national parks and preserves.  The relationships between the layers is defined in the service. When you tap on a national park `Feature` the app identifies the feature and performs a related table query then shows the annual visitors amount for the preserve. You can update the visitor amount by tapping the drop-down in the `Callout` and selecting a different amount. The app will apply the update on the server.  The color coding of the Preserves features are outlined in the Legend below. The color will change to correlate with the updated values from the app.
+# Update related features
 
-![Update Related Features App](update-related-features.png)
-![Map Legend](legend.png)
+Update related features in an online feature service.
 
-## How to use the sample
-Tap on a feature to select it. A callout will appear with a spinner which can be used to select new data. Selecting new data will also update the server.
+![Image of update related features](update-related-features.png)
+![Image of map legend](legend.png)
```

Comment:
Should the legend perhaps go in the About the data section?

Reply:
let's just leave it here, it goes with the preview image of the sample well I think.